IanGRap/Squish
# added inputs for P3 and P4

Contributors: Alfredo Cuevas, Austin Kellar

Goal:
- Allow the game to be played with 4 players

What Changed:
- We added control inputs for player 3 and player 4 in Unity's input manager
- We created a test scene in Character Selection/4 Player Test
- We updated the temp game manager to allow P3 and P4 to restart

How to Test:
- Add 4 players to any scene, set their inputs to *_P3  and *_P4 and ensure that all controls work as expected
